app/models/herbarium_record.rb

```diff
@@ -72,16 +72,16 @@ class HerbariumRecord < AbstractModel
   scope :notes_has,
         ->(str) { search_columns(HerbariumRecord[:notes], str) }
 
-  scope :initial_dets, lambda { |val|
+  scope :initial_det, lambda { |val|
     exact_match_condition(HerbariumRecord[:initial_det], val)
   }
   scope :initial_det_has,
         ->(str) { search_columns(HerbariumRecord[:initial_det], str) }
```

Comment:
Not part of PR, but interesting that this isn't covered. Haven't checked on main.

Reply:
Not covered on main.

Reply:
Yeah, this scope is of questionable utility, but its analogue is offered on the API. I can add a test there.

Reply:
This scope is already tested in `tests/query/herbarium_records_test.rb` right here:
```ruby
  def test_herbarium_record_initial_det
    expects = [herbarium_records(:field_museum_record)]
    scope = HerbariumRecord.index_order.initial_det("Lichen")
    assert_query_scope(expects, scope, :HerbariumRecord, initial_det: "Lichen")
    assert_query_scope(expects, scope, :HerbariumRecord, initial_det: "lichen")
  end
```
It seems unlikely that Coveralls is wrong, but in this case lo and behold...?

Reply:
I added a second test that covers this scope, in `test/api2`